debian/control

```diff
@@ -152,7 +152,7 @@ Package: daos-client
 Section: net
 Architecture: any
 Multi-Arch: same
-Depends: ${shlibs:Depends}, ${misc:Depends}, openmpi-bin, libfabric (>= 1.15.1-1), libfabric (<< 1.18)
+Depends: ${shlibs:Depends}, ${misc:Depends}, openmpi-bin, libfabric (>= 1.15.1-1)
```

Comment:
I think it's actually the library you want to pin here:
```suggestion
Depends: ${shlibs:Depends}, ${misc:Depends}, openmpi-bin, libfabric1 (>= 1.15.1-1)
```
or are there userspace tools that daos-server needs?

Reply:
yeah CI relies on the fi_info tools for now. In the near future I'm planning to remove the entire dependency on libfabric from DAOS server so I would not read too much into it at this point.

Reply:
Is it just CI that relies on those tools or does the DAOS product rely on them also?

Reply:
So this really should be a `Depends:` on a daos*tests* package, yes?

Reply:
right, if we wanted to do that, yes. I will make a PR relatively soon that removes the direct dependency on libfabric (and UCX) from DAOS (by making the control plane no longer directly query from them) so I would expect a lot more of simplification at that time.